config/settings/pen.yml

```
@@ -0,0 +1,41 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://sandbox.register-trainee-teachers.service.gov.uk
```

I think this needs to match the URL of the pen env, so maybe:
```suggestion
base_url: https://pen.register-trainee-teachers.service.gov.uk
```

---

Fixed